etl/app/server/database.js

```diff
@@ -421,7 +530,7 @@ export async function trimSchema(pool, s3Config) {
   }
 }
 
-// Drop old national-downloads folders
+// Get the ETL task for a particular profile
```

Comment:
Is this change intentional? It looks like the old comment is the correct description, and the new one is the same as the comment for `getProfileEtl`.

Reply:
That was accidental. I'll switch back to the original comment.